schemas/context/experienceevent.schema.json

```diff
@@ -38,9 +38,15 @@
           "description": "The time at which this interaction was received by a server."
         },
         "xdm:endUserIDs": {
+          "meta:status": "deprecated",
```

Comment:
I thought the plan was to make this a breaking change and just remove it so we're clean.

Reply:
we are going to do an initial drop of this change with both to help testing.  We will drop the older fields when we do the larger schema refactor.